### PR TITLE
Update Working here page “supporting-your-development-and-wellbeing”

### PR DESCRIPTION
**Pull request by beeLorna**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/working-here/supporting-your-development-and-wellbeing.md [hunk 1]
```diff
@@ -262,8 +262,8 @@ a number of private channels:
 
 * \#identity-lgbtqia-plus (contact a member of @identity-lgbtqia-plus-admins to
   join)
```
Comment (yndajas) [inline src/working-here/supporting-your-development-and-wellbeing.md:264]: Is it worth adding the new LGBTQIA+ group in this PR too?

**Comment by snim2** [comment]
@beeLorna has this been changed in Slack BTW?

**Comment by beeLorna** [comment]
> @beeLorna has this been changed in Slack BTW?

The user group name has been changed, just waiting for Stu to change the channel names. Can update you once this is done

**Comment by snim2** [comment]
OK, the channel name change has happened now @yndajas do you want to raise a separate PR for the other channel?

**Comment by beeLorna** [comment]
> OK, the channel name change has happened now @yndajas do you want to raise a separate PR for the other channel?

@yndajas shall we make the lgbtqia+ update til we've announced it publicy?

**Comment by yndajas** [comment]
> @yndajas shall we make the lgbtqia+ update til we've announced it publicy?

Sure yeah!

**Review by snim2 — APPROVED** [review]

**Pull request merged**